.github/workflows/ci.yml

```diff
@@ -123,8 +124,8 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python: [ '3.8', '3.9', '3.10' ]
-        airflow: [ '2.2.4', '2.3', '2.4', '2.5' ]
+        python: ['3.8', '3.9', '3.10']
```

Comment:
Add 3.11 in a separate PR

Reply:
Created ticket to track this: https://github.com/astronomer/astro-provider-databricks/issues/67